SimG4CMS/Tracker/interface/TkAccumulatingSensitiveDetector.h

```diff
@@ -46,56 +45,51 @@ public Observer<const BeginOfJob*>
     void fillHits(edm::PSimHitContainer&, const std::string&) override;
     void clearHits() override;
 
-    std::string type();
-
 private:
-    virtual void sendHit();
-    virtual void updateHit(G4Step *);
-    virtual bool newHit(G4Step *);
-    virtual bool closeHit(G4Step *);
-    virtual void createHit(G4Step *);
-    void checkExitPoint(Local3DPoint);
+    void createHit(const G4Step *);
```

Comment:
I don't understand this change? There is a derived class with its own implementation, no?

Reply:
There is no sense to have private virtual methods

Reply:
a private virtual method is just fine. But in this case nothing derives from TkAccumulatingSensitiveDetector